### ISSUE TITLE
Add basic paradigm with new styles

### ISSUE THREAD

**Issue by eddieantonio** [UAlbertaALTLab/cree-intelligent-dictionary#127]
Display basic paradigms for words with the following POS:

 - [x] VTA (verb, transitive, animate)
 - [x] NAD (noun, animate, dependent)

Using the new styles ([HTML source code](https://github.com/eddieantonio/proto-itwewina/blob/db80bed2d721057855a0d0295db3cbdfc36c63b8/src/screens/Default.svelte#L10-L40), [Preview](https://proto-itwewina.eddieantonio.now.sh)).

Then do the basic paradigms:

 - [x] VAI
 - [x] VTI
 - [x] VII
 - [x] NID
 - [x] NI
 - [X] NA

Address the following user story:

> As a Cree language learner, I want to see simplified inflectional paradigms. 

---

All other parts-of-speech do not have an inflectional table.

@aarppe: this is a good time to thinking about the exact implementation of  "paradigm panels" and how we'll make it trivial for people to upload them: perhaps straight from a spreadsheet or a CSV? Note: We will **not** require "paradigm panels" in order to close this particular issue.

**Comment by aarppe** [comment]
If the UI designer hasn't specified it yet, the Cree SRO/syllabic words in the paradigms should appear different from the labels, i.e. one should be able to easily distinguish One vs. minos, Many vs. minosak, etc. Italics?

**Comment by aarppe** [comment]
Note to self (@aarppe). We would want to shift the plain English and nêhiyawêwin paradigms to the simpler structuring (without the Algonuian division into SAP, Mixed, and Third Person (for Verbs), as user feedback has been that it makes finding the correct form difficult.) I and/or one of our linguists needs to do this, noted as a separate issue to be implemented later - as long as the new code will interpret whatever is specified in the  *,layout files.

**Comment by aarppe** [comment]
As a further follow-up note-to-self (@aarppe), as long as the pane-internal ordering will remain the same, we should be able to specify all extents of paradigm within a single data structure of the *.layout file type. I suppose we could even incorporate varying pane-internal ordering, but implementing substantially differing structures (e.g. distinguishing SAPs, Third person, and mixed participants) might turn out too complicated).

Will need to move these notes-to-selves to their own issue(s).

**Comment by Madoshakalaka** [comment]
new styles are now online. With mocked table showing "minôs" to everything.

need a brief discussion with @eddieantonio to connect the paradigm table part

![newMocked](https://user-images.githubusercontent.com/44753941/67178337-90fec780-f38f-11e9-9c9e-02e80c369f52.png)

**Comment by aarppe** [comment]
What's the TODO part standing for?

**Comment by aarppe** [comment]
And, `Ipc` = `Particle` (or `Independent Particie` in full, but that's too long).

**Comment by Madoshakalaka** [comment]
todo stands for the audio we haven't implemented yet. Use "particle" instead of "Ipc" seems like a reasonable choice. But I have to say both means nothing even to me...the former having the advantage of being a English word though

**Comment by aarppe** [comment]
For the record, the basis paradigms for independent nouns should contain the singular possessor forms (in singular) - as in the old itwêwina. On the other hand, I'm not entirely sure we'd need the diminutive forms - if we want to keep the basic paradigms simple and basic.

![image](https://user-images.githubusercontent.com/15003407/67345364-fd94d600-f4f7-11e9-8faf-f5d2c301f75d.png)

For the dependent nouns, we wouldn't have any non-possessed forms, but we'd also have the unspecified possessor.

![image](https://user-images.githubusercontent.com/15003407/67345539-9d526400-f4f8-11e9-8414-48255b96f175.png)

![image](https://user-images.githubusercontent.com/15003407/67345574-bfe47d00-f4f8-11e9-90f9-142a05486741.png)

Of course, the kinship terms do not have an unspecified possessor, so such. a block could be left entirely out (as has been discussed with entirely empty diminutive blocks).


I've been presuming that we are using the *.layout files to specify to paradigms, but if we're using some other specification source, then we need to make that explicit.

**Comment by Madoshakalaka** [comment]
@aarppe This is now online for all words! styles do need more polish, @eddieantonio needs to do some work on it

**Comment by aarppe** [comment]
Excellent! Very good start, already tested a bunch of words.

Formatting will eventually need some tweaking, and will likely benefit from multipane approach.

**Comment by eddieantonio** [comment]
This is semi-blocked by #159. I'd like to have an example of each part of speech and a regression test for each before calling this one done.

**Comment by eddieantonio** [comment]
@aarppe this has been improved a bit. Please check the latest version https://sapir.artsrn.ualberta.ca/cree-dictionary/lemma/2020215/ and tell me how it looks so far. I'd like to incorporate this feedback into integration tests after #159 is merged.

**Comment by eddieantonio** [comment]
@aarppe for conjunct forms in the basic paradigms, could we make it _one_ column right now? Two columns does not fit on mobile :(

**Comment by aarppe** [comment]
We can create mobile versions of the basic paradigms for now, having only a single column for both nouns and verbs - will get on it.

Ideally, we'd want a directory in the new _itwêwina_ source where to place these and other "new" paradigms that will be created now and on the longer term. Also ideally, I'd prefer to work with CSV layout files as the source (as currently), and even more ideally the conversion of those CSV files to whatever internal representation would be done automatically (rather than the current manual invocation of the CSV-to-TXT script).

On the longer term, for the full paradigms, I think this would be best solved with a dynamic pane approach.

- [ ] dedicated directory for paradigms being used in the new, mobile-first version of _itwêwina_
- [ ] mobile-adapted basic paradigms for all inflecting word classes

**Comment by eddieantonio** [comment]
Okay! Me and @Madoshakalaka will import directly from CSV! It'll be in a dedicated directory — probably somewhere in [res/](https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/tree/master/CreeDictionary/res).

**Comment by aarppe** [comment]
The basic/mobile paradigm layouts are now implemented as CSV files, in:

.../giella/langs/crk/inc/paradigms/mobile

These can certainly be moved under any directory more appropriate for the new _itwêwina_.

**Comment by aarppe** [comment]
For the record, are you using the new mobile paradigms, which only have a single column? These can be found in the following location within the Giella infrastructure:

https://victorio.uit.no/langtech/trunk/langs/crk/inc/paradigms/mobile/

At least in the publicly available version of the new itwêwina, the layouts have double columns.

**Comment by aarppe** [comment]
See above.

**Comment by eddieantonio** [comment]
Thanks! I'll incorporate the new layouts tomorrow.

**Comment by Madoshakalaka** [comment]
Very nice!

**Comment by eddieantonio** [comment]
> Very nice!

![Gif of Borat saying 'very nice!'](https://user-images.githubusercontent.com/2294397/70636309-8a076f00-1bf2-11ea-9c9c-a6ff4d2f1fb9.gif)
